src/graphql/error-map.ts

```diff
@@ -207,6 +208,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "Unable to find a route for payment."
       return new RouteFindingError({ message, logger: baseLogger })
 
+    case "InsufficientOnChainFundsError":
+      message = "Funds temporarily offline. Will be restored shortly."
```

Comment:
This is what we're looking for feedback on

Reply:
Alt2:
Request to send onchain funds received. Onchain replenishment initiated. Please try again in [time until request can be served]